doc/concepts/decor.rst

```diff
@@ -194,6 +202,9 @@ Currently, only parameters of :ref:`density mechanisms <cablecell-density-mechs>
     # paint a scaled density mechanism, where 'g' is scaled with the distance from the root.
     decor.paint('"dend"', arbor.scaled_mechanism(arbor.density(m), {'g': '(distance 1.0 (root))'}))
 
+    # initial value for the membrane potential
+    decor.paint('(all)', Vm=(42, '(diameter)'))
```

Comment:
Is this still correct syntax with the recent changes?

Reply:
Nope, good catch!